spec/models/user_spec.rb

```diff
@@ -31,7 +30,7 @@
   end
 
   describe ".active" do
-    it "ignores invited users" do
+    xit "ignores invited users" do
```

Comment:
xit basically marks the test as pending and doesn’t run any of the code. Need to rewrite this test and the code to join on `team_users.status = 'active'`